client/lib/desktop/index.js

```diff
@@ -117,7 +122,7 @@ var Desktop = {
 
 	onShowMySites: function() {
 		debug( 'Showing my sites' );
-		const site = this.getSelectedSite();
+		const site = this.selectedSite;
 		const siteId = this.isSingle() ? site.slug : null;
```

Comment:
Looks like this will throw too. Will fix in a bit. Seems fine to use the slug here if there's a site selected.